docs/hub/security.md

```diff
@@ -165,3 +165,18 @@ git commit -S -m "My first signed commit"
 ```
 
 Once pushed on the Hub, you should see the commit with a "Verified" badge.
+
+## Malware Scanning
+
+We run every file of your repositories through a [malware scanner](https://www.clamav.net/).
```

Comment:
```suggestion
We run every file of your public repositories through a [malware scanner](https://www.clamav.net/).
```

is it only for public repos? i don't remember

Reply:
I run it on everything if I'm correct